src/conv-core/memory-isomalloc.C

```diff
@@ -61,12 +61,10 @@ static void meta_init(char **argv)
    isomalloc_thread = 1;         /* isomalloc is allowed in this pthread */
 #endif
    if (CmiMyRank()==0) meta_inited = 1;
-#if CMK_SMP
-    if (CmiMyPe()==0 && _sync_iso == 0 && _sync_iso_warned == 0) {
+    if (CmiNumNodes() > 0 && CmiMyPe()==0 && _sync_iso == 0 && _sync_iso_warned == 0) {
         _sync_iso_warned = 1;
-        printf("Warning> Using Isomalloc in SMP mode, you may need to run with '+isomalloc_sync'.\n");
+        printf("Warning> Using Isomalloc with multiple logical nodes. You may need to run with '+isomalloc_sync'.\n");
```

Comment:
Should this be prefaced with a check on CmiNumNodes > 1? Since we don't need to warn about this in a multicore build

Reply:
Nevermind, I can't read